### PR TITLE
Input: Cleanup warnings of -Wshadow

### PR DESCRIPTION
**Pull request by howard0su**
Add prefix m_ to the fields name/ui_name/type
in ControlGroup class.

### CODE REVIEW

File: Source/Core/InputCommon/ControllerEmu/ControlGroup/ControlGroup.h [hunk 1]
```diff
@@ -89,9 +89,9 @@ class ControlGroup
     return std::copysign(std::max(T{0}, std::abs(input) - deadzone) / (T{1} - deadzone), input);
   }
 
-  const std::string name;
-  const std::string ui_name;
-  const GroupType type;
+  const std::string m_name;
+  const std::string m_ui_name;
+  const GroupType m_type;
```
Comment (stenzek) [inline Source/Core/InputCommon/ControllerEmu/ControlGroup/ControlGroup.h]: This seems inconsistent. If you're going to change the member names, it's better to update all of them.

Comment (howard0su) [reply Source/Core/InputCommon/ControllerEmu/ControlGroup/ControlGroup.h]: @BhaaLseN mentioned #8684 that he prefers to only change the fields impacted.

Comment (leoetlino) [reply Source/Core/InputCommon/ControllerEmu/ControlGroup/ControlGroup.h:94]: Using the m_ prefix feels... wrong for (public) *struct* member variables. I'd just add a _ suffix to whatever constructor arguments require them.

**Comment by leoetlino** [comment]
This seems to be obsolete; the warnings don't show up on master for me.

**Pull request closed without merging**